torch/distributed/_composable/fsdp/_fsdp_state.py

```diff
@@ -71,6 +84,16 @@ def _init_fqns(self) -> None:
             if module in module_to_fsdp_param_group:
                 module_to_fsdp_param_group[module]._module_fqn = module_name
 
+<<<<<<< HEAD
+=======
```

Comment:
will stamp once resolving this merging conflict

Reply:
Oops 😅  I fixed it now.